Assets/MRTK/Examples/Experimental/StepSlider/StepSliderExample.unity

```diff
@@ -0,0 +1,8073 @@
+%YAML 1.1
```

Comment:
In the example scene, please add a panel describing the sample scene, as in the SliderExample scene:

![image](https://user-images.githubusercontent.com/168492/92664868-0205e100-f2ba-11ea-92f3-a77a360b1e34.png)